app/views/organisations/show.html.erb

```diff
@@ -14,7 +14,7 @@
             <% row.key { attr[:name].to_s.humanize } %>
             <% row.value { details_html(attr) } %>
             <% row.action(
-              visually_hidden_text: "name",
+              visually_hidden_text: "&nbsp;" + attr[:name].to_s.humanize.downcase,
```

Comment:
Ahhhh sorry, I’m getting my different summary lists mixed up. In which case, I don’t think the `&nbsp;` is required here: 

```suggestion
              visually_hidden_text: attr[:name].to_s.humanize.downcase,
```

Reply:
yep just removed that now

Reply:
Brill, thank-you 🙂